docs/src/_guide/orchestration.md

```diff
@@ -27,6 +29,18 @@ meltano schedule add carbon__sqlite --extractor tap-carbon-intensity --loader ta
 
 Now that you've scheduled your first pipeline, you can load the "Pipeline" page in the UI and see it show up.
 
+### Scheduling predefined jobs
+
+If you've defined a [job](/reference/command-line-interface#job) within your project, you can schedule it directly without having to manually pass its plugins to the `meltano schedule add` command:
+
+```
+# Define a job
+meltano job add tap-gitlab-to-target-postgres-with-dbt --tasks "[tap-gitlab target-postgres, dbt:run]"
```

Comment:
If `--tasks "tap-gitlab target-postgres dbt:run"` would also work here (as the next doc suggests), I think we should use that as it's simpler and consistent with `meltano run`.